bootstrap/config/kfctl_basic_auth.yaml

```diff
@@ -28,7 +28,7 @@ components:
   - basic-auth
   - basic-auth-ingress
   - centraldashboard
-  - cert-manager
+  - proxy-agent
```

Comment:
You can add another entry to config folder like 
kfctl_proxy.yaml

Reply:
done